cucumber/authenticators_jwt/features/authn_jwt_fetch_signing_key.feature

```diff
@@ -569,3 +569,53 @@ Feature: JWT Authenticator - Fetch signing key
     """
     cucumber:host:myapp successfully authenticated with authenticator authn-jwt service cucumber:webservice:conjur/authn-jwt/raw
     """
+
+  Scenario: provider-uri TLS certificate validation
+    Given I load a policy:
+    """
+    - !policy
+      id: conjur/authn-jwt/keycloak
+      body:
+      - !webservice
+
+      - !variable
+        id: provider-uri
+    """
+    And I am the super-user
+    And I successfully set authn-jwt "provider-uri" variable in service "keycloak" with untrusted TLS server
```

Comment:
Can this sentence be more bright like " I set porivder-uri value to https://jwks"?